policies/postgres/policies.go

```diff
@@ -567,6 +567,35 @@ func (r policiesRepository) DeleteSinkFromAllDatasets(ctx context.Context, sinkI
 	return datasets, nil
 }
 
+func (r policiesRepository) DeleteAGroupFromAllDatasets(ctx context.Context, aGroup string, ownerID string) error {
+	q := `UPDATE datasets SET agent_group_id = null WHERE mf_owner_id = :mf_owner_id AND agent_group_id = :agent_group_id`
```

Comment:
This will make the dataset invalid though, so shouldn't we update the status to invalid here too? Or we are counting on other code to do that now?

Reply:
> This will make the dataset invalid though, so shouldn't we update the status to invalid here too? Or we are counting on other code to do that now?

Actually, this is already happening, we have only missed this deleting part